test/contacts.js

```diff
@@ -0,0 +1,172 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../app');
+
+chai.should();
+
+chai.use(chaiHttp);
+let token;
+
+describe('Contacts API', () => {
+  const user = {
+    email: 'agustin_tafura@test.com',
+    password: '123456'
+  };
+
+  before((done) => {
+    chai.request(server)
+      .post('/auth/login')
+      .send(user)
+      .end((err, response) => {
+        response.should.have.status(200);
+        token = response.body.token;
+        done();
+      });
+  });
+
+  describe('GET /backoffice/contacts', () => {
+    it('It should GET all the contacts', (done) => {
+      chai.request(server)
+        .get('/backoffice/contacts')
+        .set({ Authorization: `Bearer ${token}` })
+        .end((err, response) => {
+          response.should.have.status(200);
+          response.body.should.be.a('array');
+          response.body.map((obj) => obj.should.have.all.deep.keys('id', 'name', 'phone', 'email', 'message', 'deletedAt', 'createdAt', 'updatedAt'));
+          done();
+        });
+    });
+  });
+
+  it('it should not get the contacts, page not found', (done) => {
```

Comment:
andre aca manu nos dijo que pongamos solo "page not found" con el resto lo mismo que pongamos una frace de lo que hace el test, ej: "invalid contact",  "name required", etc

Reply:
Gracias